hub/management/commands/import_wildlife_trusts_regions.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+
+import pandas as pd
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand
+
+
+class Command(BaseImportFromDataFrameCommand):
+    help = "Import Wildlife trust reserves"
+
+    data_file = (
+        settings.BASE_DIR / "data" / "wildlife_trust_regions_in_constituencies.csv"
+    )
+    cons_row = "PCON19NM"
+    uses_gss = False
+    message = "Importing constituency Wildlife Trust regions data"
+
+    data_sets = {
+        "wildlife_trust_regions": {
+            "defaults": {
+                "label": "Wildlife Trust Regions",
+                "description": "The Wildlife Trust Region(s) that a constituency falls under.",
+                "data_type": "text",
+                "category": "place",
+                "source_label": "Data from The Widlife Trusts",
+                "source": "https://www.wildlifetrusts.org/",
+                "source_type": "csv",
+                "table": "areadata",
+                "is_filterable": True,
+                "comparators": DataSet.in_comparators(),
```

Comment:
Not sure this is the best comparator as you need to know the exact name of the trust, unless we populated the options with all the names of the trusts but 40 is possibly unwieldy for this. Might make more sense to make this a text comparator, although then can't look for more than one trust then. Maybe 40 is not that bad, although also, see other comment :|